modules/sdk/src/main/scala/org/tessellation/sdk/cli/CliMethod.scala

```diff
@@ -16,31 +16,14 @@ import org.tessellation.sdk.PriorityPeerIds
 import org.tessellation.sdk.config.types._
 
 import eu.timepit.refined.auto._
+import eu.timepit.refined.types.all.PosLong
 import fs2.io.file.Path
 
-trait CliMethod {
-
-  val keyStore: StorePath
-  val alias: KeyAlias
-  val password: Password
-
-  val environment: AppEnvironment
-
-  val seedlistPath: Option[SeedListPath]
-
-  val l0SeedlistPath: Option[SeedListPath]
-
-  val prioritySeedlistPath: Option[SeedListPath]
-
-  val stateChannelAllowanceLists: Option[Map[Address, NonEmptySet[PeerId]]]
-
-  val trustRatingsPath: Option[Path]
-
-  val httpConfig: HttpConfig
-
-  val stateAfterJoining: NodeState
-
-  val collateralAmount: Option[Amount]
+object CliMethod {
+  val snapshotSizeConfig: SnapshotSizeConfig = SnapshotSizeConfig(
+    singleSignatureSizeInBytes = 296L,
+    maxStateChannelSnapshotBinarySizeInBytes = PosLong.unsafeFrom(512_000L)
```

Comment:
Since we're constructing from a literal, the compiler can validate for us rather than at runtime. WDYT?

```suggestion
    maxStateChannelSnapshotBinarySizeInBytes = PosLong(512_000L)
```